web/php/Modules/PageRate/PageRateWidgetModule.php

```diff
@@ -52,5 +53,17 @@ public function build($runData)
             $votecount = count($rates);
             $runData->contextAdd("votes", $votecount);
         }
+
+        /**
+         * If voting is disabled, send that to the widget so we don't display a
+         * misleading widget. Used in PageRateWidgetModule.tpl
+         * @see Category::getRatingEnabled()
+         */
+        if($category->getRatingEnabled() === false) {
+            $runData->contextAdd("enabled", false);
+        }
```

Comment:
Are we able to do `$runData->contextAdd("enabled", $category->getRatingEnabled())`?

Reply:
No idea, let me see.